src/mcp_gateway/server.py

```
@@ -37,6 +39,52 @@ async def _keep_alive() -> None:
+    approval_notifier: ApprovalNotifier,
+    request: ApprovalRequest,
+    audit: AuditLogger,
+    sid: str,
+    timeout: float = 5.0,
+) -> None:
+    try:
+        await asyncio.wait_for(
+            approval_notifier.request_approval(request),
+            timeout=timeout,
+        )
+    except (asyncio.TimeoutError, Exception) as notifier_exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)